app/controllers/omniauth_controller.rb

```diff
@@ -54,6 +80,15 @@ def tra_response_excludes_trn?
   end
 
   def remove_from_get_an_identity_pilot
+    send_error_to_sentry(
+      "User removed from Pilot",
+      contexts: {
+        status: {
+          trn_blank: tra_response_excludes_trn?,
+        },
```

Comment:
I've purposefully not included any user information here, just that it's happened and that it was because they were missing a TRN. This'll provide us a count of how many people this is impacting relative to the amount of people who make it through the pilot to complete an application.